src/_includes/partials/author_bios/BobToninho.md

```diff
@@ -0,0 +1 @@
+Young freelance developer
```

Comment:
Please expand your bio here. We generally like the form “Bob Toninho is a…. He…. And I’m his spare time he….”